src/network/networkd-json.c

```diff
@@ -3,6 +3,8 @@
 #include <linux/nexthop.h>
 
 #include "dhcp-server-internal.h"
+#include "dhcp6-protocol.h"
+#include "dhcp6-lease-internal.h"
```

Comment:
`dhcp6-protocol.h` seems not necessary. At least, the two headers are not sorted.

Reply:
We need dhcp6-protocol.h as we refer to "DHCP6Message"  in dhcp6-lease-internal.h else the compilation would fail.

Reply:
I see, then please include `dhcp6-protocol.h` in `dhcp6-lease-internal.h`.

Reply:
Yes, added dhcp6-protocol.h to dhcp6-lease-internal.h